modules/ROOT/pages/sync-update-api-spec.adoc

```diff
@@ -14,7 +14,7 @@ After you update the specification, accept that Studio regenerates the flows bas
 +
 To manually generate a flow for the updated API, right-click the Mule project where you updated the API specification, and select *API Specs* > the name of your API > *Generate Flows*.
 
-Additionally, you can manage the API specifications in your project that are linked to Exchange from the API specification project management view. To do so, click the *Manage Mule Project APIs* icon (image:manage-project-api-icon.png["A stylized letter i in a house shape" 2%,2%]) in the task bar.
+Additionally, you can manage the API specifications in your project that are linked to Exchange from the API specification project management view. To do so, click the *Manage Mule Project APIs* icon (image:manage-project-api-icon.png["A stylized letter i in a house shape" 2%,2%]) in the taskbar.
```

Comment:
I missed this - I don't think you need a description for the icon here since you have the icon name right before.

Reply:
A question to consider: does the description add to the reader's understanding? I think, in this case, it might confuse...

Reply:
You didn't miss this change. I added these "taskbar" changes when you made your comments. Thanks!